code/_onclick/item_attack.dm

```diff
@@ -14,12 +14,6 @@
 		return TRUE
 	if(target.attackby(src,user, params))
 		return TRUE
-	if(QDELETED(src))
-		stack_trace("An item got deleted while performing an item attack and did not stop melee_attack_chain.")
-		return TRUE
-	if(QDELETED(target))
-		stack_trace("The target of an item attack got deleted and melee_attack_chain was not stopped.")
-		return TRUE
```

Comment:
Why are you removing this?

Reply:
moved to ifdef testing



> Why are you removing this?

I added this stack trace in a previous pr, and think it should just be during testing. It rarely makes much difference besides poor handling and has confused newer coders by throwing during unrelated prs.

Reply:
whatever, id rather get this reviewed